mods/creative/init.lua

```diff
@@ -54,7 +57,7 @@ function minetest.handle_node_drops(pos, drops, digger)
 	end
 	local inv = digger:get_inventory()
 	if inv then
-		for _, item in ipairs(drops) do
+		for _, item in pairs(drops) do
```

Comment:
This is not exactly performance-critical code...

Whether `pairs` or `ipairs` is faster also depends on implementation:

```lua
local t = {1,1,2,3,5,8,13,21,34,55}

local function timeit(f)
	local n = 0
	local start = os.clock()
	for i = 1, 1000 do
		for _, x in f(t) do
			n = n + x
		end
	end
	return os.clock()-start
end

print("ipairs:", timeit(ipairs))
print("pairs:",  timeit(pairs))
```

```shell
# luajit pairs.lua
ipairs:	0.000156
pairs:	0.000598
# lua5.1 pairs.lua
ipairs:	0.001272
pairs:	0.001225
```